.circleci/config.yml

```diff
@@ -2,6 +2,8 @@ version: 2.1
 
 orbs:
   docker: circleci/docker@0.5.20
+  jq: circleci/jq@2.2.0
+  jahia-modules-orb: jahia/jahia-modules-orb@0.1.6
```

Comment:
isn't there a more recent version we could use?

Reply:
Ahh, yeah I guess they get released pretty frequently, that was the highest version when I started but there are 3 that are more recent already :) I'll update, hopefully nothing breaks 🤞